azure-pipelines/pipelines.yml

```diff
@@ -18,45 +18,31 @@ jobs:
       inputs:
         versionSpec: '18.x'
       displayName: 'Install Node.js'
-    - script: |
-        cd ce
-
-        npm install -g @microsoft/rush
-        rc=$?; if [ $rc -ne 0 ]; then exit $rc ; fi
-
-        rush update
-        rc=$?; if [ $rc -ne 0 ]; then exit $rc ; fi
-
-        rush rebuild
-        rc=$?; if [ $rc -ne 0 ]; then exit $rc ; fi
-
-        rush test
-        rc=$?; if [ $rc -ne 0 ]; then exit $rc ; fi
-
-        rush lint
-        rc=$?; if [ $rc -ne 0 ]; then exit $rc ; fi
-
-        if [ -n "$(git status --porcelain)" ]; then
-          echo "ERROR: Working directory is dirty. Are there test output files missing from the PR?"
-          git status
-          exit 1
-        fi
-      displayName: 'Rush install, build and test vcpkg-artifacts'
+    - task: Npm@1
+      inputs:
+        command: 'custom'
+        workingDir: 'vcpkg-artifacts'
+        customCommand: 'install'
     - bash: |
-        cmake '-DCMAKE_CXX_FLAGS=-fprofile-arcs -fPIC -O0 -fsanitize=undefined -fsanitize=address' -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTING=ON -DVCPKG_DEVELOPMENT_WARNINGS=ON -DVCPKG_WARNINGS_AS_ERRORS=ON -DVCPKG_BUILD_BENCHMARKING=ON -DVCPKG_BUILD_FUZZING=ON -B build.amd64.debug
+        cmake '-DCMAKE_CXX_FLAGS=-fprofile-arcs -fPIC -O0 -fsanitize=undefined -fsanitize=address' -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTING=ON -DVCPKG_DEVELOPMENT_WARNINGS=ON -DVCPKG_WARNINGS_AS_ERRORS=ON -DVCPKG_BUILD_BENCHMARKING=ON -DVCPKG_BUILD_FUZZING=ON -DVCPKG_ARTIFACTS_DEVELOPMENT=ON -B build.amd64.debug
         make -j 2 -C build.amd64.debug
       displayName: "Build vcpkg with CMake"
       failOnStderr: true
     - bash: build.amd64.debug/vcpkg-test
-      displayName: 'Run vcpkg tests'
+      displayName: 'Run vcpkg unit tests'
       env:
         VCPKG_ROOT: UNIT_TESTS_SHOULD_NOT_USE_VCPKG_ROOT
+    - script: |
+        cd build.amd64.debug/vcpkg-artifacts
+        node node_modules/mocha/bin/mocha --config mocha-config.yaml

```

Comment:
Possibly, but we don't run a cmake configure on this node.

Reply:
Excuse me, I got my .ymls crossed. We do run a cmake configure on this node, so that could work. This form is consistent with the other test stuff though. I could see moving how tests run into the `CMakeLists.txt` but would prefer to see that done for all tests consistently unrelated to this change.